src/pages/idea-market/IdeaMarketMain.tsx

```diff
@@ -1,56 +1,152 @@
 import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
 import styles from './ideaMarketMain.module.scss';
 import PreviewThumbnail from '../../components/preview/PreviewThumbnail';
 import Dropdown from '../../components/common/dropdown/Dropdown';
 import { Carousel } from '../../components/common/carousel/Carousel';
 
-import Category from '../../assets/icons/category.svg?react';
+import UnclickBookmark from '../../assets/icons/unclickBookmark.svg?react';
+import Bookmark from '../../assets/icons/bookmark.svg?react';
+
+interface CardData {
+  id: number;
+  isBookmarked: boolean;
```

Comment:
아마 선택적 prop으로 두면, 나중에 prop 값을 넣을 때, 조금 더 직관적으로 보일 것 같습니다.

```suggestion
  isBookmarked?: boolean;
```